src/components/PopoverMenu/PopoverMenuItem.vue

```diff
@@ -100,6 +101,15 @@ export default {
 			return this.item.key
 				? this.item.key
 				: Math.round(Math.random() * 16 * 1000000).toString(16)
+		},
+		iconIsUrl() {
+			try {
+				// eslint-disable-next-line no-unused-vars
```

Comment:
if you don't use the object I guess you can just remove the `const url =` part and it should still do the same